src/containers/BooksList.js

```diff
@@ -2,10 +2,12 @@ import React from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import Book from '../components/Book';
+import { removeBook } from '../actions/index';
 
-function BooksList({ books }) {
+function BooksList({ books, deleteBook }) {
   const bookList = books.map(book => (
     <Book
+      deleteBook={deleteBook}
```

Comment:
- [ ] While I know that passing the `deleteBook` action directly as a prop to Book will work, let's try to follow React conventions and create a function that handles the deletion, using the delete action, named `handleRemoveBook` as instructed at Pathwright:
![ImplementsRemoveBook](https://user-images.githubusercontent.com/16193212/94950780-8f071900-04b9-11eb-855d-e7df0b4b7699.png)

Imagine if you would like to make a little confirm modal or alert show up every time you click `Remove book`. If you just pass the action to the Book component that becomes tricky, but if you have the `handleRemoveBook` function calling the action it's very easy to add a couple more of lines of code inside it to add the extra effects you want to happen.

Reply:
Hi Sarah :wave: , thanks for explaining why it could useful to keep a `handleRemoveBook` method. I did not think about the utility of having it for extra effects. :sunglasses: